ocaml/xapi/storage_migrate.ml

```diff
@@ -821,6 +823,9 @@ let copy ~task ~dbg ~sr ~vdi ~dp ~url ~dest =
       let similars = List.map (fun vdi -> vdi.content_id) similar_vdis in
       debug "Similar VDIs to %s = [ %s ]" vdi (String.concat "; " (List.map (fun x -> Printf.sprintf "(vdi=%s,content_id=%s)" x.vdi x.content_id) vdis));
```

Comment:
This log message actually seems wrong, I think we should be listing `similars` here. :thinking:

Reply:
Indeed, from 4521c61 we can clearly see that the message originally printed the VDIs returned by `Local.VDI.similar_content`

Reply:
There is another place where the log is wrong, in `let start'`

Reply:
Raised PR https://github.com/xapi-project/xen-api/pull/3252 to fix the logs.